### PR TITLE
1040 rate limiting tests + new limits

### PR DESCRIPTION
**Pull request by thomasyopes**
Ticket: #2986 

### Description

- add artillery tests
- adjust limits out the door

### Testing

- Local
  - [x] artillery tests work against staging

### Release Plan

- [ ] Merge this

### CODE REVIEW

File: packages/artillery-tests/src/fhir/batch/payload-functional.json
Comment (thomasyopes) [inline packages/artillery-tests/src/fhir/batch/payload-functional.json:1]: I ran npm prettier on this file..... but didn't edit anything

File: packages/artillery-tests/.env.example [hunk 1]
```diff
@@ -5,4 +5,8 @@ MEDPLUM_PASSWORD=password
 MEDPLUM_URL=https://domain.com
 
 HAPI_URL=https://domain.com
-HAPI_TENANT_ID=some-id
\ No newline at end of file
+HAPI_TENANT_ID=some-id
+
+OSS_STAGING_URL=https://domain.com
+OSS_STAGING_API_KEY=some-key
+OSS_STAGING_FACILITY_ID=some-id
```
Comment (leite08) [inline packages/artillery-tests/.env.example:12]: Why "staging" on the env var name? We represent that with diff `.env` files, one for each env.

File: packages/artillery-tests/src/rate-limiting/patient/staging/document-query.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # https://www.artillery.io/docs/guides/plugins/plugin-publish-metrics
+    publish-metrics:
+      - type: cloudwatch
+        region: "{{ $processEnvironment.REGION }}"
+    # https://www.artillery.io/docs/guides/plugins/plugin-metrics-by-endpoint
+    metrics-by-endpoint:
+      # Group metrics by request name rather than URL:
+      useOnlyRequestNames: true
+  phases:
+    # Tested at 10 operatiosn / min
```
Comment (leite08) [inline packages/artillery-tests/src/rate-limiting/patient/staging/document-query.yml:15]: typo: operations

**Review by leite08 — APPROVED** [review]
Since we're repurposing this, I'd also update the README to account for the fact this package now is for general artillery tests, how they're organized, etc. Cool to do that on a next PR so we can ship the updates on the infra and code, though, so approving just in case.

**Pull request merged**